provisionersdk/archive.go

```diff
@@ -96,6 +96,10 @@ func Tar(w io.Writer, directory string, limit int64) error {
 			// Don't store tfstate!
 			return nil
 		}
+		if rel == "terraform.tfvars" || rel == "terraform.tfvars.json" || strings.HasSuffix(rel, ".auto.tfvars") || strings.HasSuffix(rel, ".auto.tfvars.json") {
+			// Don't store .tfvars, as Coder uses their own variables file.
+			return nil
+		}
```

Comment:
It would be nice if we could log any ignored paths; we only use this in one place so it shouldn't be a huge change.

Reply:
Done. I introduced `logger.Debug`, so it does not scramble the default CLI output.